### PR TITLE
Attempt to Streamline UI Tasks

### PR DESCRIPTION
**Pull request by andyasberry**
These changes are being added to help streamline the UI portion.

### CODE REVIEW

File: tasks-ui.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Tasks
+
+ - [ ] Clean up App.js (5 mins)
+ - [ ] Add styling to Storage Summary (5-10 mins)
+ - [ ] Dry up StorageSummary component (10 mins)
+ - [ ] Issue PR (you can use `git publish` on this machine)
```
Comment (andyasberry) [inline tasks-ui.md]: Maybe I should remove the part in parentheses here, since it is specific to a git alias on my machine?

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+registry=http://nexus.containerstore.com/nexus/repository/npm-group/
```
Comment (nikb747) [inline .npmrc]: @andyasberry I don't think we should add this, limits the usability of this test

File: tasks-ui.md (hunk 1 above)
Comment (andyasberry) [reply tasks-ui.md]: I'm going to take care of this

File: .npmrc (hunk 2 above)
Comment (andyasberry) [reply .npmrc]: I need to remove this and will today.

File: tasks-ui.md (hunk 1 above)
Comment (andyasberry) [reply tasks-ui.md]: Done👍

File: .npmrc (hunk 2 above)
Comment (andyasberry) [reply .npmrc]: Done 👍

**Pull request merged**